docs/tutorials/gluon/fit_api_tutorial.md

```diff
@@ -0,0 +1,311 @@
+
+
+```python
+transformers = [gluon.data.vision.transforms.Resize(224), # We pick 224 as the model we use takes an input of size 224.
+                gluon.data.vision.transforms.ToTensor(), 
+                gluon.data.vision.transforms.Normalize(mean = 0, std = 1)]
+
+# Now we will stack all these together.
+transform = gluon.data.vision.transforms.Compose(transformers)
+```
+
+
+```python
+# Apply the transformations
+fashion_mnist_train = fashion_mnist_train.transform_first(transform)
+fashion_mnist_val = fashion_mnist_val.transform_first(transform)
+```
+
+## Data Loaders
+
+In order to feed the data to our model, we need to use a [Data Loader](https://mxnet.incubator.apache.org/api/python/gluon/data.html?highlight=dataloader#mxnet.gluon.data.DataLoader). 
+
+Data Loaders are used to read the dataset, create mini-batches from the dataset and feed them to the neural network for training.
+
+
+```python
+batch_size = 256 # Batch size of the images
+num_workers = 4 # The number of parallel workers for loading the data using Data Loaders.
+
+train_iter = gluon.data.DataLoader(fashion_mnist_train, batch_size=batch_size, shuffle=True, num_workers=num_workers)
+val_iter = gluon.data.DataLoader(fashion_mnist_val, batch_size=batch_size, shuffle=False, num_workers=num_workers)
+```
+
+## AlexNet Model
+
+AlexNet architecture rose to prominence when it won the [2012 ImageNet LSVRC-2012 competition](http://image-net.org/challenges/LSVRC/2012/).
+
+It contains 5 convolutional layers and 3 fully connected layers. Relu is applied after very convolutional and fully connected layer. Dropout is applied before the first and the second fully connected year.
+
+The [Gluon CV Model Zoo](https://gluon-cv.mxnet.io/model_zoo/classification.html#imagenet) contains a rich collection of state-of-the-art pre-trained models for Computer Vision related tasks.
+
+We will use the ```get_model()``` API from Gluon CV Model Zoo to load the network architecture. 
+
+
+```python
+alexnet_model = get_model('alexnet', pretrained=False, classes = 10, ctx=ctx)
+```
+
+## Initialize the model parameters
+
+
+```python
+alexnet_model.initialize(force_reinit=True, init = mx.init.Xavier(), ctx=ctx)
+```
+
+## Loss Function, Trainer and Optimizer
+
+After defining the model, let's setup the trainer object for training. 
+
+We will be using ```SoftmaxCrossEntropyLoss``` as the loss function since this is a multi-class classification problem. We will be using ```sgd``` (Stochastic Gradient Descent) as the optimizer. 
+
+
+```python
+loss_fn = gluon.loss.SoftmaxCrossEntropyLoss()
+learning_rate = 0.04 # You can experiment with your own learning rate here
+
+trainer = gluon.Trainer(alexnet_model.collect_params(), 
+                        'sgd', {'learning_rate': learning_rate})
+```
+
+## Metrics to Capture
+
+We will be most interested in monitoring the accuracy here. Let's define the metrics for that.
+
+
+```python
+train_acc = mx.metric.Accuracy()
```

Comment:
How about if user wants to monitor multiple metric? You can provide a samples on how to add multiple metric. Thanks!

Reply:
Okay. Will add that.

Reply:
Let's keep this tutorial simple for now. We can add it in a separate advanced user tutorial later on.